.github/workflows/lock-closed-issues.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Lock inactive closed issues'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  lock:
+    name: "Lock"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Lock"
+        uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ github.token }}
+          issue-lock-inactive-days: '31'
+          issue-exclude-created-before: ''
+          issue-exclude-labels: ''
+          issue-lock-labels: ''
+          issue-lock-comment: >
```

Comment:
Will I get emails for every old issue?

Reply:
Overall this could be more harmful than useful.

Reply:
@othercorey You can turn off notification emails and just use web notifications :)

But I agree even that would be annoying. I'll check if it's possible to make it not generate notifications but I doubt that's possible. Will just close the PR in that case.

Reply:
Does locking without comment generate notification?

Reply:
If it can lock without comment notification that seems ok.

Reply:
maybe if you could set a starting day that would be cool...

Reply:
Ah!.. isn't this the option `issue-exclude-created-before`? So maybe old issues could stay as they are and have a starting day.. that would ease the pain of notifications...

Reply:
There is `issue-exclude-created-before` to exclude very old issues

Reply:
Event without the comment GH will send notification to at least the author about issue/PR closure. That's not good either.

Reply:
If it's just the author, that might not be so bad.

Reply:
Yes.. the problem I believe is with old issues.. if you set a starting date i believe it will be fine...

Reply:
Still, we are not solving any issues, only create them.
The 1 per year issue would be on the other side of constant noise of this closing.
I don't think that is worth it. Here it suffices to copy and paste a quick message to reopen new.

In fact, in some cases it is even warranted to just add to the existing.
So by forcing this we might even more shoot ourselves int the foot than necessary.

Reply:
I say it was a nice idea, but maybe we try again when there are new options.

Reply:
Have we confirmed that locking will send notifications?

Reply:
* Locking without comment doesn't seem to generate notifications hence I have removed the comment.
* Updated config to lock both issues and PRs.
* Issues/PRs prior to 1st May 2020 won't be processed.

We can give it a try and if it does create annoyance we can always get rid of it.

Reply:
mark and I tested. Did not receive notifications just from locking.

Reply:
I am still against this. If a PR got closed and could be opened, this would be blocked unnecessarily
and same for tickets - it creates more fuss and noise where nothing needed to be done IMO.

I see that in the phpstan repo right now, getting tons of noise for nothing.